openupgrade_scripts/scripts/stock_account/17.0.1.1/upgrade_analysis_work.txt

```diff
@@ -0,0 +1,31 @@
+---Models in module 'stock_account'---
+---Fields in module 'stock_account'---
+stock_account / account.move.line        / cogs_origin_id (many2one)     : NEW relation: account.move.line
+# NOTHING TO DO: new feature
+
+stock_account / stock.move               / analytic_account_line_id (many2one): DEL relation: account.analytic.line
+stock_account / stock.move               / analytic_account_line_ids (many2many): NEW relation: account.analytic.line
```

Comment:
It isn't a replacement of analytic_account_line_id by analytic_account_line_ids ?